packages/react-devtools-shared/src/devtools/views/Components/InspectedElementHooksTree.js

```diff
@@ -28,6 +27,7 @@ import {
 } from 'react-devtools-feature-flags';
 import HookNamesModuleLoaderContext from 'react-devtools-shared/src/devtools/views/Components/HookNamesModuleLoaderContext';
 import isArray from 'react-devtools-shared/src/isArray';
+import {copyToClipboard} from 'react-devtools-shared/src/backend/utils';
```

Comment:
Since this is inside a view, can you move it out from react-devtools-shared/src/backend/utils to react-devtools-shared/src/utils?